.github/workflows/ci.yaml

```diff
@@ -0,0 +1,43 @@
+name: tests
+on:
+  push:
```

Comment:
Does this mean that it will run on every push to any branch? If so, is it possible to get it to run only on PRs? https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-pull_request-workflow-based-on-the-head-or-base-branch-of-a-pull-request

Reply:
according to the docs, my change will have it run tests any time a PR is opened, reopened, or has its head updated.